tufrepo/cli.py

```diff
@@ -152,22 +152,34 @@ def init_succinct_roles(ctx: Context, role: str):
 
 @cli.command()
 @click.pass_context
-@click.option("--target-in-repo/--no-target-in-repo", default=True)
-@click.option("--follow-delegations/--no-follow-delegations", default=True)
-@click.option("--role", default="targets")
+@click.option("--no-target-in-repo", is_flag=True, default=False)
```

Comment:
What benefits do we get from using `is_flag`?
Yes, it's clearer what to expect when you read the code, but is there something else?
I ask you as the `click` documentation is not really clear about this.

Reply:
this is a good question: I'll try if it works without.

Reply:
If you don't have either `is_flag` or the "--on/--off" format  the option is assumed to have an argument (like `--no-target-in-repo BOOLEAN`) which I didn't want.

So I think it is correct now.

Reply:
Ahaaa, okay, I get it.